### PR TITLE
MGTransferGlobalCoarseningTools::create_geometric_coarsening_sequence load forest directly

### PR DESCRIPTION
**Pull request by peterrum**
As discussed in https://github.com/dealii/dealii/pull/11707#pullrequestreview-585109577.

FYI and thanks: @marcfehling 

Depends on #11707

### CODE REVIEW

File: include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h [hunk 1]
```diff
@@ -1620,6 +1622,105 @@ namespace internal
...
+    template <int spacedim>
+    void
+    load_forest(const parallel::distributed::Triangulation<1, spacedim> &in,
+                parallel::distributed::Triangulation<1, spacedim> &      out)
+    {
+      (void)in;
+      (void)out;
+    }
+  } // namespace internal
+
```
Comment (kronbichler) [inline include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h:1652]: add two empty lines

**Review by kronbichler — APPROVED** [review]
OK once rebased on top of #11707.

File: include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h (hunk 1 above)
Comment (peterrum) [reply include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h:1652]: I'll add them once I rebase.

**Comment by peterrum** [comment]
I have updated this PR. But GitHub seems to have some problems right now to be in sync with the branches.

**Pull request merged**